elide-datastore/elide-datastore-aggregation/pom.xml

```diff
@@ -44,10 +44,36 @@
             <groupId>com.yahoo.elide</groupId>
             <artifactId>elide-core</artifactId>
         </dependency>
+
+        <!-- Test -->
+        <dependency>
+            <groupId>com.yahoo.elide</groupId>
+            <artifactId>elide-integration-tests</artifactId>
```

Comment:
Remove this dependency.  We definitely don't want to run the elide-integration-tests on this store.